docs/architecture-and-design.md

```diff
@@ -386,6 +389,17 @@ Most [Intents](#intents) are easy to break down into  [Primitives](#primitives),
 
 In order for a robot to move to the desired destination of a `MoveIntent`, the Navigator will use various path-planning algorithms to find a path across the field that does not collide with any robots or violate any restrictions set on the `MoveIntent`. The Navigator then translates this path into a series of `MovePrimitives`, which are sent to the robot sequentially so that it follows the planned path across the field.
 
+### Path Manager
+The `Path Manager` is responsible for generating a set of paths that don't collide. It is given a set of path objectives and path planner, and it will generate paths using the given path planner and arbitrate between paths to prevent collisions. One approach is to add extra obstacles to cause the path planner to generate paths that avoid potential collisions; this is what `VelocityObstaclePathManager` implements. One disadvantage of this approach is that it does not account for when robots will occupy particular positions on the path. (Trajectory planning or use of the Minkowski space are possible ideas to account for time.)
+
+### Path Planner
+The `Path Planner` is responsible for path planning a single robot around a single set of obstacles from a given start to a given destination. Note that the interface of the path planner is intentionally decoupled from the rest of AI, so that it doesn't rely on the representation of [Intents](#intents) or [World](#world).
+
+#### Theta Star Path Planner
```

Comment:
This seems like it would be more appropriate as in-code documentation? That way it's more coupled to the code. IMO the main benefit about these big docs are architecture / design / interfaces, not necessarily specific implementations.

Reply:
That's a really good point, I agree. I should have thought of this earlier :slightly_frowning_face:.

Reply:
made the discussion more about the architecture and design

Reply:
I think any explanation of Theta* really belongs in the code, so could we convert this to a class jdoc comment for the class (and move it accordingly).

Reply:
This still needs to be addressed

Reply:
ok moved documentation to the header file